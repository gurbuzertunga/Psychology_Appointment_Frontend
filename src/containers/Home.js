import { GrDebian } from 'react-icons/gr';
import React from 'react';
import profilePic from '../assets/profilepic.jpg';

const Home = () => (
  <div className="flex flex-col lg:mt-1 mt-32">
    <div className="flex flex-col items-center">
      <GrDebian className="text-white text-8xl mt-4" />
      <h1 className="text-white  lg:text-2xl sm:text-4xl text-2xl font-bold mt-4">Jennifer Porter</h1>
      <p className="text-white font-light lg:text-base sm:text-2xl text-base mt-4">Clinical Psychologist, Psychotherapist, MSc</p>
    </div>
    <div className="flex sm:flex-row flex-col-reverse justify-evenly items-center  lg:mt-4 sm:mt-72 mt-24">
      <div className="flex flex-col sm:w-6/12 w-10/12">
        <h1 className="text-white lg:text-2xl sm:text-4xl text-2xl font-bold">ABOUT ME</h1>
        <p className="text-white lg:text-base sm:text-2xl text-base font-light mt-4">
          Lorem ipsum dolor, sit amet consectetur adipisicing elit.
          Excepturi facere, provident ipsa commodi officia fugiat harum,
          rerum assumenda delectus sapiente soluta omnis exercitationem in dicta?
        </p>
      </div>
      <img className="rounded-full sm:w-2/12 w-4/12" src={profilePic} alt="Profile" />
    </div>
  </div>
);

export default Home;
