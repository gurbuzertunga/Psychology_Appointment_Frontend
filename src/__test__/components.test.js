import renderer from 'react-test-renderer';
import {
  BrowserRouter,
} from 'react-router-dom';
import LoginForm from '../components/loginForm';
import SignUpForm from '../components/signUpForm';
import Home from '../containers/Home';
import NotFound from '../containers/404';

it('should render the home page correctly', () => {
  const homePage = renderer.create(<Home />).toJSON;
  expect(homePage).toMatchSnapshot();
});

it('should render the home page correctly', () => {
  const homePage = renderer.create(<Home />).toJSON;
  expect(homePage).toMatchSnapshot();
});

it('should render the 404 page correctly', () => {
  const notFound = renderer.create(
    <BrowserRouter>
      <NotFound />
    </BrowserRouter>,
  ).toJSON;
  expect(notFound).toMatchSnapshot();
});

it('should render LoginForm correctly', () => {
  const loginForm = renderer.create(<LoginForm handleClick={(() => true)} />).toJSON();
  expect(loginForm).toMatchSnapshot();
});

it('should render signupForm correctly', () => {
  const signUpForm = renderer.create(<SignUpForm handleClick={(() => true)} />).toJSON();
  expect(signUpForm).toMatchSnapshot();
});
