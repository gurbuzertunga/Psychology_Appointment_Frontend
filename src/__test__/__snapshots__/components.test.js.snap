// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoginForm correctly 1`] = `
<form
  className="flex flex-col border border-gray-300 rounded py-4 mx-auto w-5/12"
  onSubmit={[Function]}
>
  <input
    className="focus:outline-none focus:ring focus:border-blue-300 bg-transparent bolder placeholder-white font-light text-white p-1 mb-4 mx-6 border border-gray-300 rounded"
    id="email"
    onChange={[Function]}
    placeholder="Email"
    type="email"
    value=""
  />
  <input
    className="focus:outline-none focus:ring focus:border-blue-300 bg-transparent bolder placeholder-white font-light text-white p-1 mb-4 mx-6 border border-gray-300 rounded"
    id="password"
    onChange={[Function]}
    placeholder="Password"
    type="password"
    value=""
  />
  <input
    className="focus:outline-none focus:ring focus:border-blue-300 bg-transparent bolder text-white mr-2 border w-4/12 place-self-center font-light border-gray-300 rounded"
    type="submit"
    value="Submit"
  />
</form>
`;

exports[`should render signupForm correctly 1`] = `
<form
  className="flex flex-col border border-gray-300 rounded py-4 mx-auto w-5/12"
  onSubmit={[Function]}
>
  <input
    className="font-light focus:outline-none focus:ring focus:border-blue-500 bg-transparent bolder placeholder-white text-white p-1 mb-4 mx-6 border border-gray-300 rounded"
    id="name"
    onChange={[Function]}
    placeholder="Name"
    type="text"
    value=""
  />
  <input
    className="font-light focus:outline-none focus:ring focus:border-blue-500 bg-transparent bolder placeholder-white text-white p-1 mb-4 mx-6 border border-gray-300 rounded"
    id="email"
    onChange={[Function]}
    placeholder="Email"
    type="email"
    value=""
  />
  <input
    className="font-light focus:outline-none focus:ring focus:border-blue-500 bg-transparent bolder placeholder-white text-white p-1 mb-4 mx-6 border border-gray-300 rounded"
    id="password"
    onChange={[Function]}
    placeholder="Password"
    type="password"
    value=""
  />
  <input
    className="font-light focus:outline-none focus:ring focus:border-blue-500 bg-transparent bolder placeholder-white text-white p-1 mb-4 mx-6 border border-gray-300 rounded"
    id="passwordConfirm"
    onChange={[Function]}
    placeholder="Password Confirm"
    type="password"
    value=""
  />
  <input
    className="font-light focus:outline-none focus:ring focus:border-blue-500 bg-transparent bolder text-white mr-2 border w-4/12 place-self-center border-gray-300 rounded"
    type="submit"
    value="Submit"
  />
</form>
`;

exports[`should render the 404 page correctly 1`] = `[Function]`;

exports[`should render the home page correctly 1`] = `[Function]`;

exports[`should render the home page correctly 2`] = `[Function]`;

exports[`should render the welcome page correctly 1`] = `[Function]`;
